src/app/static/src/app/store/language/actions.ts

```diff
@@ -24,8 +24,9 @@ export const ChangeLanguageAction = async (context: ActionContext<RootState, Roo
 
     const actions: Promise<unknown>[] = [];
 
-    if (rootState.baseline?.iso3 && rootGetters["baseline/complete"] && rootGetters["surveyAndProgramme/complete"]) {
```

Comment:
So I noticed that this was never actually firing because of a typo in "surveyAndProgramme" it should be "surveyAndProgram"... oops :see_no_evil: 

But this wasn't needed anyway as we refetch the metadata every time ReviewInput component is mounted. So this will always be refetched anyway